Rocket/Rocket/RocketDetail/Components/ParametersSectionView.swift

```diff
@@ -14,29 +14,29 @@ struct ParametersSectionView: View {
     let mass: Int
     
     var body: some View {
-        VStack {
-            
-            // Parameters title
-            HStack {
-                Text("Parameters")
-                    .font(.headline)
-                    .padding(.leading)
-                
-                Spacer()
-            }
+        VStack(alignment: .leading) {
             
             // Parameters info
             HStack {
-                ParameterView(name: "height", number: Int(height), unit: "m")
+                ParameterView(
+                    name: "height",
+                    number: Int(height),
+                    unit: "m")
```

Comment:
Bavili sme sa na iOS meetingu, že by sme chceli zachovávať jednotné formátovanie, tak by asi bolo fajn použiť to, čo máme v My2NApp. Myslím, že tam máme nastavený page guide na 120 chars a zalamujeme iba ak to pretečie cez tú čiaru 🙂 
S nastavením toho guidu keď tak bude vedieť pomôcť @Finemas